testpackage/tests/Magnetosphere_3D_small/Magnetosphere_3D_small.cfg

```diff
@@ -115,6 +115,7 @@ constBgBX = 0.0
 constBgBY = 0.0
 constBgBZ = -5.0e-9
 noDipoleInSW = 0 # Use 0 with dipole type 4
+dipoleScalingFactor = 0.1 # smaller dipole for faster testing
```

Comment:
Don't tell Minna. :P

Reply:
Now runs to 6 seconds instead of 3 seconds
```
Magnetosphere_3D_small
------------------------------------------------------------
 ref-time     |   new-time       |  speedup                |
------------------------------------------------------------
95.35        40.63         2.34679
------------------------------------------------------------
```
up for discussion of course.

Reply:
I think it's OK. This is not a realistic m'sphere setup anyway. If/when we develop a larger-scale testpackage I suppose we should stick closer to production-like parameters.

Reply:
Yep, I'm ok with it too. The main reason we run the testpackage is to get all solvers to play with some semi-realistic numbers, and to check whether we broke anything.
Proper physical validation of the code should be a bit of an orthogonal concept, which we might also want to formalize in some way at some point.

But not in this PR. :)